shared/test/unit/mocks/mock_l10n.js

```diff
@@ -8,7 +8,7 @@
     get: stringify,
 
     formatValue: function(id, args) {
-      return Promise.resolve(stringify(id, args));
+      return Promise.resolve(stringify(id, args || ''));
```

Comment:
I actually know now - if args is undefined then it gets `tostringed` in stringify and then

`{
  id: 'foo',
  args: undefined
}`

gets turned into `fooundefined` instead of `foo`. Keep it :)